bulma/bulma.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from importlib import import_module
+
+from django import VERSION as DJANGO_VERSION
+from django.conf import settings
+
+
+# Do we support set_required and set_disabled?
+# See GitHub issues 337 and 345
+# TODO: Get rid of this after support for Django 1.8 LTS ends
+DBS3_SET_REQUIRED_SET_DISABLED = DJANGO_VERSION[0] < 2 and DJANGO_VERSION[1] < 10
```

Comment:
This comes from django-bootstrap I guess, we don't use it

Reply:
yes, use django-bootstrap
my website  change from django-bootstrap to bulma